Decred Wallet/AppDelegate.swift

```diff
@@ -31,6 +31,9 @@ class AppDelegate: UIResponder {
     
     var reachability: Reachability!
     
+    // We only want one instance of `SyncManager()` through out the lifecycle of the app accessible through AppDelegate.shared
+    var syncManager = SyncManager.shared
```

Comment:
Why can't we just call `SyncManager.shared` from wherever we need to access the instance? Makes it easier to know where the features we're using are defined. Accessing it from AppDelegate means we'd have to first come here to inspect this declaration before recognizing that we're dealing with a shared instance of the SyncManager class.

Reply:
resolved in 2c74553